examples/fonts.rs

```diff
@@ -30,7 +30,7 @@ pub fn main() {
 
 fn ui_system(mut ctx: IcedContext<UiMessage>) {
     ctx.display(column!(
-        text(format!("I am the default font")).font(font::Font::DEFAULT),
-        text(format!("I am another font"))
```

Comment:
Are there any technical reasons to prefer `to_string` over `format!`?

Reply:
By using format you are overriding the specialized to_string function. A general good practice in this case would be to use to_string unless you need some extra formatting

Reply:
By using format you are overriding the specialized to_string function. A general good practice in this case would be to use to_string unless you need some extra formatting